modules/ROOT/pages/migration-munit-assert-processor-changes.adoc

```diff
@@ -0,0 +1,143 @@
+= Migrating Assert Processors (MUnit)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+All assert event processors change of name space to 'assert'. +
```

Comment:
either: 
All assert event processors change**d their** name space to 'assert'. +
or
All assert event processors **had a** change of name space to 'assert'. +

Reply:
done via suggestion

Reply:
done via suggestion